mne/io/tests/test_constants.py

```diff
@@ -0,0 +1,260 @@
+# Author: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import re
+import shutil
+import zipfile
+
+from mne.io.constants import FIFF
+from mne.utils import _fetch_file, requires_good_network
+
+
+# These are oddities that we won't address:
+iod_dups = (355, 359)  # these are in both MEGIN and MNE files
+tag_dups = (3501, 3507)  # in both MEGIN and MNE files
+# The tests should also be improved, see several XXX below.
+
+_dir_ignore_names = ('clear', 'copy', 'fromkeys', 'get', 'items', 'keys',
+                     'pop', 'popitem', 'setdefault', 'update', 'values')
+
+# XXX These should all probably be added to the FIFF constants
+_missing_names = (
+    'FIFFV_BEM_APPROX_CONST',
+    'FIFFV_BEM_APPROX_LINEAR',
+    'FIFFV_COIL_ARTEMIS123_GRAD',
+    'FIFFV_COIL_ARTEMIS123_REF_GRAD',
+    'FIFFV_COIL_ARTEMIS123_REF_MAG',
+    'FIFFV_COIL_BABY_REF_MAG',
+    'FIFFV_COIL_BABY_REF_MAG2',
+    'FIFFV_COIL_KRISS_GRAD',
+    'FIFFV_COIL_POINT_MAGNETOMETER_X',
+    'FIFFV_COIL_POINT_MAGNETOMETER_Y',
+    'FIFFV_COIL_SAMPLE_TMS_PLANAR',
+    'FIFFV_NEXT_SEQ',
+    'FIFFV_NEXT_NONE',
+    'FIFFV_PROJ_ITEM_DIP_FIX',
+    'FIFFV_PROJ_ITEM_DIP_ROT',
+    'FIFFV_PROJ_ITEM_FIELD',
+    'FIFFV_PROJ_ITEM_HOMOG_FIELD',
+    'FIFFV_PROJ_ITEM_HOMOG_GRAD',
+    'FIFFV_PROJ_ITEM_NONE',
+    'FIFF_UNIT_AM_M2',
+    'FIFF_UNIT_AM_M3',
+    'FIFFV_MNE_COORD_4D_HEAD',
+    'FIFFV_MNE_COORD_CTF_DEVICE',
+    'FIFFV_MNE_COORD_CTF_HEAD',
+    'FIFFV_MNE_COORD_FS_TAL',
+    'FIFFV_MNE_COORD_FS_TAL_GTZ',
+    'FIFFV_MNE_COORD_FS_TAL_LTZ',
+    'FIFFV_MNE_COORD_KIT_HEAD',
+    'FIFFV_MNE_COORD_MNI_TAL',
+    'FIFFV_MNE_COORD_MRI_VOXEL',
+    'FIFFV_MNE_COORD_RAS',
+    'FIFFV_MNE_COORD_TUFTS_EEG',
+)
```

Comment:
@agramfort this is the to-do list for `fiff-constants` now that I have added tests for everything, I cannot find these in that repo :(

But I think it should just require a few more `enum`s in the `fiff-constants` to get most of these.

Reply:
ok